desktop/src/main/start.ts

```diff
@@ -104,20 +117,19 @@ function hookOpenWindow (window: BrowserWindow): void {
     } else {
       void (async (): Promise<void> => {
         const bounds = mainWindow?.getBounds()
-        const childWindow = new BrowserWindow({
+        const windowOptions: Electron.BrowserWindowConstructorOptions = {
           width: bounds?.width ?? defaultWidth,
           height: bounds?.height ?? defaultHeight,
           x: (bounds?.x ?? 0) + 25,
           y: (bounds?.y ?? 0) + 25,
-          titleBarStyle: isMac ? 'hidden' : 'default',
```

Comment:
Should we change setupWindowTitleBar to {...getWindowStyle()} to prevent possible glitch or inability to change style on the fly?